src/datanode/src/heartbeat/handler.rs

```diff
@@ -32,7 +33,12 @@ pub struct HeartbeatResponseHandlerContext {
     pub mailbox: MailboxRef,
     pub response: HeartbeatResponse,
     pub incoming_message: Option<IncomingMessage>,
-    is_skip_all: bool,
+}
+
+#[derive(PartialEq)]
```

Comment:
Adds some comments to `HandleControl` and `HeartbeatResponseHandler` trait APIs.